css/style.css

```diff
@@ -85,3 +85,334 @@ header {
+  align-items: flex-start;
+  justify-content: space-around;
+  width: 329px;
+  height: 555px;
+  background-color: #fff;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+  padding: 16px;
+}
+
+.card-section-text {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 12px;
+}
+
+.heading-title {
+  color: #172b4d;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+}
+
+.card-sub-container {
+  display: flex;
+  align-items: center;
+  gap: 12px;
+}
+
+.title-canopy {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.title-developer {
+  font-family: "Poppins", sans-serif;
+  color: #7a869a;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.title-year {
+  color: #7a869a;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+}
+
+.desciption-p-mobile {
+  color: #344563;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+}
+
+.pro-language {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 15px;
+  list-style-type: none;
+}
+
+.html-1 {
+  background: #ebebff;
+  border-radius: 8px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 19px;
+  padding: 4px 12px;
+  color: #6070ff;
+  letter-spacing: 0.03;
+}
+
+.btn-see-project {
+  color: #396df2;
+  border: 1px solid #396df2;
+  border-radius: 8px;
+  width: 124px;
+  height: 48px;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  padding: 5px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  background-color: #fff;
+}
+
+.about-section {
+  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 114px 21px;
+  gap: 24px;
+  border-radius: 0 80px 0 0;
+}
+
+.about-section-inside {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 50px;
+  gap: 12px;
+}
+
+.about-heading {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 40px;
+  line-height: 52px;
+  background-color: #fff;
+  width: 327px;
+  height: 101px;
+  display: flex;
+  align-items: center;
+  padding-bottom: 12px;
+}
+
+.about-description {
+  font-family: "Poppins", sans-serif;
+  width: 327px;
+  height: 144px;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: #344563;
+}
+
+.connect-link1 {
+  width: 120px;
+  height: 24px;
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 16px;
+  line-height: 24px;
+  color: #7f8cff;
+}
+
+.social-media {
+  display: flex;
+  list-style-type: none;
+  gap: 19px;
+}
```

Comment:
- [ ] Kindly make sure to center the social media links in the container in order to look like the provided Figma design :wink: 

   - ### YOUR DESIGN
   
     ![image](https://user-images.githubusercontent.com/87197412/212264446-d1a2ffd5-abb0-46c9-a2b7-36dc9d065596.png)

   - ### FIGMA DESIGN
   
    ![image](https://user-images.githubusercontent.com/87197412/212264522-27574be3-ca59-454a-867b-e27c00ec8014.png)